primedev/plugins/interfaces/sys/ISys.cpp

```diff
@@ -0,0 +1,66 @@
+#include "plugins/interfaces/interface.h"
+#include "ISys.h"
+#include "plugins/plugins.h"
+#include "plugins/pluginmanager.h"
+
+class CSys : public ISys
+{
+public:
+	void Log(HMODULE handle, LogLevel level, char* msg)
+	{
+		spdlog::level::level_enum spdLevel;
+
+		switch (level)
+		{
+		case LogLevel::WARN:
+			spdLevel = spdlog::level::level_enum::warn;
+			break;
+		case LogLevel::ERR:
+			spdLevel = spdlog::level::level_enum::err;
+			break;
+		default:
+			NS::log::PLUGINSYS->warn("Attempted to log with invalid level {}. Defaulting to info", (int)level);
+		case LogLevel::INFO:
+			spdLevel = spdlog::level::level_enum::info;
+			break;
+		}
```

Comment:
No `critical` level? ig we dont rly use it in northstar so whatever

Reply:
is the critical level even used once in northstar

Reply:
dont think so, but its still a thing that exists, don't think it really matters if it's an option here